internal/cmd/kafka/command_mirror_failover.go

```diff
@@ -14,10 +14,11 @@ import (
 
 func (c *mirrorCommand) newFailoverCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "failover <destination-topic-1> [destination-topic-2] ... [destination-topic-N] --link my-link",
-		Short: "Failover mirror topics.",
-		Args:  cobra.MinimumNArgs(1),
-		RunE:  c.failover,
+		Use:               "failover <destination-topic-1> [destination-topic-2] ... [destination-topic-N] --link my-link",
```

Comment:
We've never included flags in the usage string... maybe we should do this for all commands though, since it's required?
```suggestion
		Use:               "failover <destination-topic-1> [destination-topic-2] ... [destination-topic-N]",
```

Reply:
Nah I don't think it's necessary. I'll just commit your suggestion.